extensions/adobe/experience/adcloud/conversiondetails.example.1.json

```diff
@@ -1,6 +1,6 @@
 {
-  "https://ns.adobe.com/experience/adcloud/conversiondetails/attributedEventType": "imp",
-  "https://ns.adobe.com/experience/adcloud/conversiondetails/conversionObjective": "click_to_req",
-  "https://ns.adobe.com/experience/adcloud/conversiondetails/eventUid": "09877uy898ukajsdlnc",
-  "https://ns.adobe.com/experience/adcloud/conversiondetails/conversionMetric": "array<struct<conversion_model:string,ct_count:double,ct_value:double,vt_count:double,vt_value:double>>"
+  "https://ns.adobe.com/experience/adcloud/attributedEventType": "imp",
+  "https://ns.adobe.com/experience/adcloud/conversionObjective": "click_to_req",
+  "https://ns.adobe.com/experience/adcloud/eventUid": "09877uy898ukajsdlnc",
+  "https://ns.adobe.com/experience/adcloud/conversionMetric": "array<struct<conversion_model:string,ct_count:double,ct_value:double,vt_count:double,vt_value:double>>"
```

Comment:
I don't understand the value here - it isn't a standard XDM value

Reply:
Removing conversiondetails schema from PR for further internal review.